observability-kit-agent/src/main/java/com/vaadin/extension/VaadinObservabilityInstrumentationModule.java

```diff
@@ -53,9 +51,6 @@
         extends InstrumentationModule {
 
     static {
```

Comment:
I suppose the empty static block can be removed

Reply:
Leftover, removed 👍